pkg/build/builder/util.go

```diff
@@ -71,6 +71,10 @@ func getDockerNetworkMode() s2iapi.DockerNetworkMode {
 func GetCGroupLimits() (*s2iapi.CGroupLimits, error) {
 	byteLimit, err := readInt64("/sys/fs/cgroup/memory/memory.limit_in_bytes")
 	if err != nil {
+		// for systems without cgroups builds should succeed
+		if _, err := os.Stat("/sys/fs/cgroup"); os.IsNotExist(err) {
+			return &s2iapi.CGroupLimits{}, nil
+		}
```

Comment:
did you check how docker ends up handling these default values?  it's going to set the memory limit to 0 bytes, and cpu shares/period/quota as well.  does docker treat 0s for those as unlimited?